packages/core/src/preview.ts

```diff
@@ -208,15 +208,15 @@ export const getPreview = (
         executing.push({
           referenceName: 'GnosisSafe',
           functionName: 'deploy',
-          variables: [],
+          variables: {},
```

Comment:
Changed this in a bunch of places to b/c the default structure of the `decodedActions.variables` is an object instead of an array as of the [previous PR](https://github.com/sphinx-labs/sphinx/pull/1401). (This is because we use the `recursivelyConvertResult` function)